<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraDialog</class>
 <widget class="QDialog" name="CameraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Total : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCount">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>32</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CameraDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
